src/script/calling/CallingRepository.ts

```diff
@@ -698,7 +708,18 @@ export class CallingRepository {
       return 0;
     }
 
-    const options = this.targetMessageRecipients(payload, destinationUserId, destinationClientId);
+    let options: MessageSendingOptions;
+
+    if (typeof targets === 'string') {
+      const parsedTargets: SendMessageTarget = JSON.parse(targets);
```

Comment:
`JSON.parse` might throw. I don't see that handled.

Reply:
It's an async callback which cannot be fulfilled when `JSON.parse` throws which is okay since we then get notified about the error.